client/state/sites/selectors.js

```diff
@@ -150,9 +151,10 @@ export const getSiteCollisions = createSelector(
  * @param  {Number}  siteId Site ID
  * @return {Boolean}        Whether collision exists
  */
-export function isSiteConflicting( state, siteId ) {
-	return includes( getSiteCollisions( state ), siteId );
-}
+export const isSiteConflicting = treeSelect(
+	( state, siteId ) => [ getSiteCollisions( state, siteId ) ],
+	( [ siteCollisions ], siteId ) => includes( siteCollisions, siteId )
```

Comment:
How big the `siteCollisions` array typically is? In my account, I don't have any conflicting sites and the array is empty. In other accounts, there may be one or two conflicts? Calling `includes` on such a small array is very cheap and it's not worth memoizing. I wouldn't be suprised if the `treeSelect` overhead actually made it a bit slower 🐌

Reply:
Ah, that's a great point. For some reason, I thought it's iterating over all the sites.